### ISSUE TITLE
How to contribute to the website?

### ISSUE THREAD

**Issue by mipmip** [protoncr/tourmaline#54]
It's not clear how to send PR's for the website. There is a tourmaline-website repo and this repo has a gh-pages branch. I see multiple typo's in de code examples or examples that do not work anymore.

It would help if this was clearly documented and if the website-repo is outdated it would be wise to remove it.

Thanks

**Comment by watzon** [comment]
Sorry it took me a while to get to this, I stepped away from Crystal and Telegram for a while. As far as contributing to the website: 

This project uses [mkdocstrings-crystal](https://mkdocstrings.github.io/crystal/index.html), so you'll need install the dependencies with `pip install -r requirements.txt` first.

You'll then need to make sure you have shard depencies installed using `shards install`.

Run `mkdocs build` to build the files, and `mkdocs serve` to start a local development server so you can test your changes.

Then all you have to do is commit and push. Once your PR is merged into master a Github action will run that builds the docs and pushes them to the `gh_pages` branch.
